Naamgeving regels

sanderke opened the pull request.

fterpstra committed:
Update transport-security.md

added descriptive identifiers & how to test

joostfarla commented on transport-security.md:
```diff
@@ -13,16 +13,18 @@ Note: security controls for signing and encrypting of application level messages
-  <p class="rulelab"><strong>API-11</strong>: Secure connections using TLS</p>
+<span id="api-11"></span>
+<div class="rule" id="/transport/TLS">
+  <p class="rulelab"><strong>/transport/TLS</strong>: Secure connections using TLS</p>
```
De standaard voor rule-IDs is `kebab-case`. Deze zouden dus altijd lowercase moeten zijn.

sanderke committed:
kebab-case

https://github.com/Logius-standaarden/API-mod-transport-security/pull/1#discussion_r1615665562
